.travis.yml

```diff
@@ -1,32 +1,29 @@
-language: ruby
-sudo: required
+language: generic
 env:
   CI: yes
-rvm:
-- 2.5
+  DB_USERNAME: travis
+  DB_PASSWORD: root
 services:
   - docker
+  - mysql
+  - redis
 before_install:
   - mkdir -p ~/.aws && printf "[default]\nregion = $AWS_REGION\naws_access_key_id = $AWS_ACCESS_KEY_ID\naws_secret_access_key = $AWS_SECRET_ACCESS_KEY\n" > ~/.aws/credentials
-  - bin/build-docker $TRAVIS_BRANCH ${TRAVIS_COMMIT::8} $TRAVIS_BUILD_NUMBER
-  - docker-compose up -d db
-  - docker-compose up -d redis
-  - sleep 10
-  - docker-compose run web rake db:create
-install:
-  - gem install brakeman
+  - bin/build-docker $TRAVIS_BRANCH ${TRAVIS_COMMIT::8}
+  # map db and redis to localhost to seamlessly use travis-ci services instead of docker-compose
+  - sudo bash -c "echo '127.0.0.1 db' >> /etc/hosts"
+  - sudo bash -c "echo '127.0.0.1 redis' >> /etc/hosts"
```

Comment:
Also, is this relevant? https://docs.travis-ci.com/user/hosts/ Possibly a better way.

Reply:
Yes! Thanks much better!